### ISSUE TITLE
[BUG] Regression: seekTo 0 with pause not working on iOS

### ISSUE THREAD

**Issue by themadmrj** [jhomlala/betterplayer#569]
The code was working with v0.0.69, not working anymore with v0.0.70
I'll skip the template for regressions if you don't mind. I found the issue while integrating the iOS preCache feature in the new code.
Basically, when my video finishes, I want to reset it, which was working fine with the following code:
```
    _betterPlayerController!.addEventsListener((event) {
      if (event.betterPlayerEventType == BetterPlayerEventType.finished) {
        //reset video

        _betterPlayerController
            ?.seekTo(Duration())
            .then((value) => _betterPlayerController?.pause());
      }
    }
```

This is working on Android (v0.0.70), not on iOS. I tried pausing and THEN using seekTo(0), doesn't work either. The result on iOS is that seekTo is executed but pausing is ignored.

**Comment by jhomlala** [comment]
@themadmrj This will be fixed with next release.

**Comment by jhomlala** [comment]
Fixed in 0.0.71.

**Comment by nguyenxuankiet262** [comment]
I'm using 0.0.83 but it's not working
